pepper-apis/src/main/java/org/broadinstitute/ddp/model/copy/CompositeCopyConfigurationPair.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.ddp.model.copy;
+
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
+
+public class CompositeCopyConfigurationPair {
+
+    private long id;
+    private long copyConfigPairId;
+    private long sourceChildQuestionStableCodeId;
+    private String sourceChildQuestionStableId;
+    private long targetChildQuestionStableCodeId;
+    private String targetChildQuestionStableId;
+
+    @JdbiConstructor
```

Comment:
Consider getting rid of dumb warning.
```suggestion
    @SuppressWarnings("unused")
    @JdbiConstructor
```

Reply:
Is this an Intellij IDE warning? I don't see anything... I'll refrain from adding lines unless it's required.

Reply:
IntelliJ compiler warning. I find them distracting to see repeatedly and have to decide multiple times if they are real or just that compiler not smart enough to figure out that it is called via reflection. Up to you.